samples/CommunityToolkit.Maui.Sample/Pages/Views/MediaElement/MediaElementPage.xaml.cs

```diff
@@ -114,12 +114,11 @@ void OnMuteClicked(object? sender, EventArgs e)
 		MediaElement.ShouldMute = !MediaElement.ShouldMute;
 	}
 
-	void BasePage_Unloaded(object? sender, EventArgs e)
+	protected override void OnDisappearing()
```

Comment:
Please, do not use `OnAppearing/Disappearing` use the `OnNavigatedTo/From` instead (: